backend/run.json

```diff
@@ -1,11 +1,11 @@
 {
   "scripts": {
-    "start": "deno run --no-check --allow-read --allow-net --config deno.json --watch src/app.ts",
-    "build": "deno compile --config deno.json src/app.ts -o dist/jimmi",
-    "lint": "deno lint --config deno.json",
-    "fmt": "deno fmt --config deno.json",
-    "test": "deno test --config deno.json",
-    "cov": "deno coverage --config deno.json"
+    "start": "deno run     --config deno.json --import-map=../import_map.json --no-check --allow-read --allow-net --watch src/app.ts",
```

Comment:
I think we have to update the api client generation pipeline as well to use the new command and import map

Reply:
Fixed with 9bd0441cc1e7f29e28f5691241d4e75ad532c4aa